style.css

```diff
@@ -8,15 +8,28 @@
             width: 100%;
         }
         
-        .menu {
-            width: 80%;
+        .part{
+            width:100%;
+        }
+
+        .limit{
+            width:80%;
```

Comment:
Zostawiłem width: 80%, ale może lepiej zamienić to na max-width:[wartość w px], ponieważ przy zmianie szerokości ekranu szerokość klasy limit będzie zawsze 80%, a tak to doszłoby do iluśtam px i już odstępów by nie było. Podobnie chyba zrobiliśmy na Food is Good.